web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
```

Comment:
Should we add a line like this:

https://github.com/OCA/OCB/blob/12.0/addons/web/static/src/js/fields/basic_fields.js#L1785

Reply:
@KKamaa is this done

Reply:
what exactly was to be added in this line of code?

Reply:
supportedFieldTypes: ['many2one'],